consensus/hotstuff/persister/persister.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 // Persister can persist relevant information for hotstuff.
+// Persister depends on protocol.State root snapshot bootstrapping to retrieve chainID.
```

Comment:
```suggestion
// Persister depends on protocol.State root snapshot bootstrapping to set initial values for
// SafetyData and LivenessData. These values must be initialized before first use of Persister.
```

The initializing of the values was more what I meant.